.travis/before-install-osx.sh

```diff
@@ -2,12 +2,22 @@
 
 export HOMEBREW_NO_ANALYTICS=1
 
-brew install libsndfile || brew install libsndfile || exit 1
+if $UPDATE_HOMEBREW; then
```

Comment:
also add `export HOMEBREW_NO_INSTALL_CLEANUP=1` in all builds, this prevents homebrew from doing an expensive cleanup step rarely ("brew clean has not been run in X days, running now")